src/OtherPages/login.jsx

```diff
@@ -1,28 +1,43 @@
-import React from 'react';
+import React from "react";
 import "../styles/login.css";
+import { authenticationProvider } from "../providers";
 
 const LogComponent = (props) => {
-    const { isLogin, onClickCallback } = props;
-    const title = isLogin ? 'Log in your account' : 'Sign up';
-    const buttonText = isLogin ? 'Log in' : 'Sign up';
-    const emailRef = React.createRef();
-    const passwordRef = React.createRef();
+  const { isLogin } = props;
+  const title = isLogin ? "Log in your account" : "Sign up";
+  const buttonText = isLogin ? "Log in" : "Sign up";
+  const emailRef = React.createRef();
+  const passwordRef = React.createRef();
 
-    const _handleClick = () => {
-        return onClickCallback(emailRef.current.value, passwordRef.current.value);
+  const _handleClick = async () => {
+    const email = emailRef.current.value;
+    const password = passwordRef.current.value;
+
+    if (isLogin) {
```

Comment:
login and signup functionality, this doesn't do much yet, but you can look on the your browser's localStorage to see user and token information.

Saving an access token on the localStorage isn't the most secure way, I'm using this by convenience, if you want to change it to a httpOnly cookie, do it.

Reply:
any reference to learn this concept ?

Reply:
You can learn something from this post: https://www.bezkoder.com/react-refresh-token/
But seeing the nature of the application, saving the access token on the local storage is totally ok, there nothing much an attacker could even if he could hijack this token.